Updating MCP docs based on feedback (#8137)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/integrate-mcp-claude.md

## Claude Code

> You can set up Claude Code with both the local and remote `dbt-mcp` server. We recommend using the local `dbt-mcp` for more developer-focused workloads.

Reviewer: should we also then say when to use remote? i'm confused myself

> ### Setup with local dbt MCP server

Reviewer 2: ```suggestion
### Set up with local dbt MCP server
```

Prerequisites:

> - Completed the [local MCP setup](/docs/dbt-ai/setup-local-mcp)

Reviewer 2: ```suggestion
- Complete the [local MCP setup](/docs/dbt-ai/setup-local-mcp).
```

> - Know your configuration method (OAuth, dbt Core or Fusion, or environment variables)

Reviewer: ```suggestion
- Know your configuration method (OAuth <Constant name="dbt_core"/> or <Constant name="fusion"/>, or environment variables)
```

> - Know your configuration method (OAuth, dbt Core or Fusion, or environment variables)

Reviewer 2: ```suggestion
- Know your configuration method (for example, OAuth, <Constant name="core" /> or <Constant name="fusion_engine" />, or environment variables).
```

#### Using OAuth or environment variables directly

> The preferred method is to configure environment variables directly in Claude Code's configuration file without needing a separate `.env` file:

Reviewer 2: ```suggestion
The recommended method is to configure environment variables directly in Claude Code's configuration file without needing a separate `.env` file:
```

```bash

> # First, add the MCP server

Reviewer 2: I suggest to number these steps so it's clearer, for example:

1. Add the MCP server.
2. Edit the configuration to add environment variables.
3. In the configuration editor, add your environment variables based on your use case:

claude mcp add dbt -- uvx dbt-mcp

# Then edit the configuration to add environment variables
claude mcp edit dbt
```

In the configuration editor, add your environment variables based on your use case:

<Tabs>
<TabItem value="CLI only">

> For <Constant name="core" /> or <Constant name="fusion" /> only (no dbt platform):

Reviewer 2: ```suggestion
For <Constant name="core" /> or <Constant name="fusion" /> only (no <Constant name="dbt_platform" />):
```

> For <Constant name="core" /> or <Constant name="fusion" /> only (no dbt platform):

Reviewer: i haven't tested it but where are they adding these env vars? in the settings.json file?

```json
{
  "command": "uvx",
  "args": ["dbt-mcp"],
  "env": {
    "DBT_PROJECT_DIR": "/path/to/your/dbt/project",
    "DBT_PATH": "/path/to/your/dbt/executable"
  }

> }

Reviewer 3: Calling out that this is a subtly different JSON format than what we have on the website/docs/docs/dbt-ai/setup-local-mcp.md page. I'm not sure of the best way of resolving that.

## Claude Code

You can set up Claude Code with both the local and remote `dbt-mcp` server. We recommend using the local `dbt-mcp` for more developer-focused workloads. See the [About MCP](/docs/dbt-ai/about-mcp#server-access) page for more more information about local and remote server features.

> ### Set up with local dbt MCP server

Reviewer: looks like this section is just about setting env var - can we make the title more specific?

Author: Moving this down further

> 

Reviewer: Sugg having an intro sentence that clarifies the diff ways to config env vars. elg.

You can configure env var in a few diff ways:
- oauth (linnk to oauth)
- env file (link to env section)

## Claude for desktop

> 1. Go to the Claude settings. Click on the Claude menu in your system's menu bar (not the settings within the Claude window itself) and select **Settings…**

Reviewer 2: ```suggestion
1. Go to the Claude settings. Click on the Claude menu in your system's menu bar (not the settings within the Claude window itself) and select **Settings…**.
```

2. In the Settings window, navigate to the **Developer** tab in the left sidebar. This section contains options for configuring MCP servers and other developer features.
3. Click the **Edit Config** button and open the configuration file with a text editor.

> 4. Add your server configuration based on your use case. Choose the [correct JSON structure](https://modelcontextprotocol.io/quickstart/user#installing-the-filesystem-server) from the options below:

Reviewer 2: ```suggestion
4. Add your server configuration based on your use case. Choose the [correct JSON structure](https://modelcontextprotocol.io/quickstart/user#installing-the-filesystem-server) from the following options:
```

    <Expandable alt_header="Local MCP with OAuth">

    #### Local MCP with dbt platform authentication <Lifecycle status="managed, managed_plus" />

    Configuration for users who want seamless OAuth authentication with the <Constant name="dbt_platform" />

    <MCPExample />

    </Expandable>

    <Expandable alt_header="Local MCP (CLI only)">

>     Local configuration for users who only want to use dbt CLI commands with <Constant name="Core" /> or <Constant name="fusion" />

Reviewer 2: ```suggestion
    Local configuration for users who only want to use dbt CLI commands with <Constant name="core" /> or <Constant name="fusion" />
```
